tests/test_integration_working.py

```
@@ -18,7 +18,7 @@ def is_api_running():
     try:
         response = requests.get("http://localhost:8000/health", timeout=2)
         return response.status_code == 200
-    except:
+    except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)